cmd/attestation-container/attestation-container.go

```diff
@@ -86,66 +97,94 @@ func (s *server) FetchAttestation(ctx context.Context, in *pb.FetchAttestationRe
 
 func validateFlags() {
 	if *platformCertificateServer != "" && *platformCertificateServer != "AMD" && *platformCertificateServer != "Azure" {
-		log.Fatalf("invalid --platform-certificate-server value %s (valid values: 'AMD', 'Azure')", *platformCertificateServer)
+		logrus.Fatalf("invalid --platform-certificate-server value %s (valid values: 'AMD', 'Azure')", *platformCertificateServer)
 	}
 }
 
 func main() {
 	flag.Parse()
+
+	// if logFile is not set, logrus defaults to stderr
+	if *logFile != "" {
+		// If the file doesn't exist, create it. If it exists, append to it.
+		file, err := os.OpenFile(*logFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0666)
```

Comment:
We really wouldn't want the group or other to be able to write to the log.  `0644` for a log file at most.